package.json

```diff
@@ -12,7 +12,9 @@
     "genDataItems": "node ./database/ItemDataGenerationScript.js",
     "genDataReviews": "node ./database/ReviewDataGenerationScript.js",
     "genDataColors": "node ./database/ColorDataGenerationScript.js",
-    "genDataSizes": "node ./database/SizesDataGenerationScript.js"
+    "genDataSizes": "node ./database/SizesDataGenerationScript.js",
+    "creatdb": "psql postgres -U postgres < ./database/postgreSQL/schema.sql",
+    "seedpg": "psql postgres -U postgres -d product_options < ./database/postgreSQL/seedpg.sql"
```

Comment:
you can make these 2 into a single script using && in between them. not codebreaking or anything but an easy optimization for future users